app/webpacker/components/Tickets/TicketWorkbenches/AnonymizationTicketWorkbenchForWrt/PerformManualChecks.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { List, ListItem } from 'semantic-ui-react';
+
+export default function PerformManualChecks() {
+  // TODO: Add necessary links in this page.
```

Comment:
Is this TODO supposed to be here? Or did you just forget to take care of it before opening the PR?